it/controllers/CalculationControllerISpec.scala

```diff
@@ -450,21 +656,103 @@ class CalculationControllerISpec extends IntegrationSpecCommonBase {
+        parsedBody.select("#main-content .govuk-summary-list__row").get(2).select("dd").text() shouldBe "£277.00"
+        parsedBody.select("#main-content .govuk-summary-list__row").get(3).select("dt").text() shouldBe "Amount left to pay"
+        parsedBody.select("#main-content .govuk-summary-list__row").get(3).select("dd").text() shouldBe "£123.00"
+        parsedBody.select("#main-content div .govuk-warning-text").text() shouldBe "! This penalty will rise to £246.00 (a further 2% of the unpaid VAT) if you do not make a VAT payment by 7 April 2021."
+        parsedBody.select("#main-content .govuk-body").get(0).text() shouldBe "Paying part of your VAT bill will reduce further penalties."
+        parsedBody.select("#main-content .govuk-body").get(1).text() shouldBe "Penalties and interest will show as estimates if HMRC has not been given enough information to calculate the final amounts."
+        parsedBody.select("#main-content h2").text() shouldBe "Estimates"
+        parsedBody.select("#main-content a").get(0).text() shouldBe "Return to VAT penalties and appeals"
+        parsedBody.select("#main-content a").get(0).attr("href") shouldBe "/penalties"
+      }
+
+      "return 200 (OK) and render the view correctly when the user has specified a valid penalty ID (only one interest charge)" in new Setup(true) {
+        returnPenaltyDetailsStub(penaltyDetailsWithDay15Charge)
+        val request = controller.onPageLoad("12345678901234", false)(fakeRequest)
+        status(request) shouldBe Status.OK
+        val parsedBody = Jsoup.parse(contentAsString(request))
+        parsedBody.select("#main-content .govuk-summary-list__row").get(1).select("dt").text() shouldBe "Calculation"
+        parsedBody.select("#main-content .govuk-summary-list__row").get(1).select("dd").text() shouldBe "2% of £123.00 (VAT amount unpaid on 23 March 2021)"
+      }
+
+      "return 200 (OK) and render the view correctly with Penalty Amount" in new Setup(true) {
+        returnPenaltyDetailsStub(penaltyDetailsWithDueDateMoreThan30days)
+        val request = controller.onPageLoad("12345678901234", false)(fakeRequest)
+        status(request) shouldBe Status.OK
+        val parsedBody = Jsoup.parse(contentAsString(request))
+        parsedBody.select("#main-content .govuk-summary-list__row").get(0).select("dt").text() shouldBe "Penalty amount"
+        parsedBody.select("#main-content .govuk-summary-list__row").get(0).select("dd").text() shouldBe "£400.00"
+      }
+
+      "return 500 (ISE) when the user specifies a penalty not within their data" in new Setup(true) {
+        returnPenaltyDetailsStub(samplePenaltyDetails)
+
+        val request = controller.onPageLoad("1234567890", false)(fakeRequest)
+        status(request) shouldBe Status.INTERNAL_SERVER_ERROR
+      }
+
+      "return 300 (SEE_OTHER) when the user is not authorised" in new Setup(true) {
```

Comment:
Could you name the parameter in the Setup class + change status code
```suggestion
      "return 303 (SEE_OTHER) when the user is not authorised" in new Setup(true) {
```